pkgs/development/python-modules/dash/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, plotly
+, flask
+, flask-compress
+, future
+, dash-core-components
+, dash-renderer
+, dash-html-components
+, dash-table
+}:
+
+buildPythonPackage rec {
+  pname = "dash";
+  version = "1.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1i6a406z9law9j4151mxvips2n2j5mjkpv0smzml16gb299hxcz4";
+  };
+
+  propagatedBuildInputs = [
+    plotly
+    flask
+    flask-compress
+    future
+    dash-core-components
+    dash-renderer
+    dash-html-components
+    dash-table
+  ];
+
+  # No tests in archive
+  doCheck = false;
+
+  meta = {
+    description = "Dash is a Python framework for building analytical web applications. No JavaScript required.";
+    homepage = https://dash.plot.ly/;
+    license = with lib.licenses; [ mit ];
```

needs other suggestion to work
```suggestion
    license = licenses.mit;
```

---

Done in https://github.com/NixOS/nixpkgs/commit/411a351773bb6b47beb51f1530f5510aea376dbd